From d5973fdb1f525222599de6c343215f0c604f6c74 Mon Sep 17 00:00:00 2001
From: Jeremy Cline <jcline@redhat.com>
Date: Thu, 25 Mar 2021 09:29:08 -0400
Subject: [PATCH] Update test fixture keys to 4096 bits

The text fixture key was 1024 bits, which tools are beginning to be
upset about as they're not particularly secure. Generate a new test
key pair which should be good until RSA is broken.

Fixes #231
Signed-off-by: Jeremy Cline <jcline@redhat.com>
---
 fedora_messaging/tests/fixtures/cert.pem | 172 +++++++++++++++--------
 fedora_messaging/tests/fixtures/key.pem  |  64 +++++++--
 2 files changed, 160 insertions(+), 76 deletions(-)

diff --git a/fedora_messaging/tests/fixtures/cert.pem b/fedora_messaging/tests/fixtures/cert.pem
index 6b202ec..6d0b60f 100644
--- a/fedora_messaging/tests/fixtures/cert.pem
+++ b/fedora_messaging/tests/fixtures/cert.pem
@@ -1,76 +1,124 @@
 Certificate:
     Data:
         Version: 3 (0x2)
-        Serial Number: 1 (0x1)
-        Signature Algorithm: sha1WithRSAEncryption
-        Issuer: C=US, ST=NC, L=Raleigh, O=Fedora Project, OU=fedmsg, CN=fedmsg/name=fedmsg/emailAddress=admin@fedoraproject.org
+        Serial Number:
+            31:22:32:64:15:4f:34:e4:6a:5b:2e:c0:30:a7:1b:7a:54:38:07:84
+        Signature Algorithm: sha256WithRSAEncryption
+        Issuer: C = US, ST = NC, L = Raleigh, O = Fedora Project, OU = fedmsg, CN = fedmsg, name = fedmsg, emailAddress = admin@fedoraproject.org
         Validity
-            Not Before: Jul 15 21:18:52 2012 GMT
-            Not After : Jul 13 21:18:52 2022 GMT
-        Subject: C=US, ST=NC, L=Raleigh, O=Fedora Project, OU=fedmsg, CN=fedmsg/name=fedmsg/emailAddress=admin@fedoraproject.org
+            Not Before: Mar 25 13:26:16 2021 GMT
+            Not After : Mar 23 13:26:16 2031 GMT
+        Subject: C = US, ST = NC, L = Raleigh, O = Fedora Project, OU = fedmsg, CN = fedmsg, name = fedmsg, emailAddress = admin@fedoraproject.org
         Subject Public Key Info:
             Public Key Algorithm: rsaEncryption
-                Public-Key: (1024 bit)
+                RSA Public-Key: (4096 bit)
                 Modulus:
-                    00:b1:a2:5c:d7:10:6a:83:a4:1d:65:66:51:80:f5:
-                    72:2d:15:af:77:92:ad:1d:ad:80:d8:c8:e2:ee:cf:
-                    10:3b:07:02:1e:4f:18:c4:47:d6:82:8d:53:ff:c2:
-                    1b:72:49:fe:62:fe:68:26:f0:91:8f:d7:e0:9e:b8:
-                    46:96:ef:ef:4b:07:f4:21:10:09:d9:90:92:a6:25:
-                    3a:29:ca:b6:e8:69:06:df:ba:a7:73:b1:71:c3:6c:
-                    1c:d3:7d:d6:ff:3a:ce:15:b7:bc:7b:fc:6c:6b:d6:
-                    6f:a7:fd:e0:28:59:07:49:bf:66:9c:3e:1f:5f:b8:
-                    85:c4:61:51:f9:67:00:40:a7
+                    00:d1:6d:94:e7:6f:04:43:21:e5:05:90:6c:e7:eb:
+                    d1:6d:ea:ee:bf:dd:0d:f4:c7:e7:da:a5:d9:97:0d:
+                    4c:9e:2a:4c:c4:ff:b6:19:3a:a9:9b:19:50:01:60:
+                    eb:72:82:a9:90:13:a9:32:b8:82:03:70:3f:99:6c:
+                    8f:de:bb:58:17:37:26:de:98:49:ed:ba:df:25:4b:
+                    14:d9:c1:2b:f2:21:8b:b4:01:2f:e4:e1:49:e5:ce:
+                    2e:6f:5a:b5:c3:5f:e7:09:c1:f7:67:5b:e5:4b:8d:
+                    6c:b3:b2:94:da:91:af:c3:07:bb:16:0d:34:ee:bc:
+                    c2:55:d0:68:df:ce:e4:81:4f:74:f2:3b:57:7a:5d:
+                    99:17:75:d1:c8:c0:90:fb:2f:c4:9b:86:c8:94:8c:
+                    c2:d5:d5:d7:2b:6a:c8:3e:49:88:4c:6c:52:53:0a:
+                    99:fd:64:1a:10:8b:e1:90:26:f5:ab:d9:da:f2:2e:
+                    7d:ac:e8:7a:06:f9:29:88:0b:b9:f8:ad:b5:8c:44:
+                    c2:65:66:56:86:07:89:9e:2c:62:d1:a5:c0:fa:79:
+                    4e:4f:6b:c7:e2:8d:c4:23:b7:83:f9:cc:e6:e6:08:
+                    7c:70:f1:6c:6c:46:b5:9c:a4:2f:d2:7c:0e:2e:e4:
+                    f6:74:53:a6:9a:5f:20:92:3b:bd:34:f4:89:96:06:
+                    3b:25:94:cd:51:9f:2c:b7:8f:4c:47:0e:8a:80:56:
+                    e4:d5:57:52:23:36:72:57:06:1c:10:4d:69:e2:92:
+                    55:ff:cd:2e:ef:52:c1:31:21:b7:fa:41:c7:f2:57:
+                    bd:71:de:46:eb:94:45:40:3f:2a:1e:2f:76:b9:a3:
+                    6c:2b:2c:65:6b:b7:5a:8a:b7:d0:0c:43:05:06:10:
+                    5e:40:39:d1:93:b8:51:7c:57:07:57:ea:6b:75:94:
+                    6c:6d:0e:a0:e3:40:b8:bf:bb:0a:36:eb:22:42:88:
+                    25:a7:90:d4:f6:0f:92:e0:05:23:f9:93:09:1c:44:
+                    35:14:d7:64:bb:79:f1:66:60:8e:9a:6b:0f:d8:ff:
+                    19:93:9b:c9:95:16:86:79:8b:04:89:ff:2c:77:74:
+                    d1:dd:58:67:08:fe:c8:a9:13:c1:59:ba:17:61:62:
+                    76:63:b5:e8:47:96:33:98:ce:3d:9c:27:6a:9b:5d:
+                    e4:8a:9c:d5:31:89:07:45:41:f3:b5:9a:1b:56:e2:
+                    85:36:ef:dc:4f:b9:67:80:34:0c:49:77:f8:f0:47:
+                    b3:86:6e:68:33:3d:55:0c:0f:f0:28:9f:f6:46:98:
+                    16:37:c0:37:2e:a5:7c:6f:33:ae:00:50:df:5a:39:
+                    f5:80:ce:ea:39:60:20:9b:98:ad:30:7b:14:e1:14:
+                    4d:a8:77
                 Exponent: 65537 (0x10001)
         X509v3 extensions:
-            X509v3 Basic Constraints: 
-                CA:FALSE
-            Netscape Cert Type: 
-                SSL Server
-            Netscape Comment: 
-                Easy-RSA Generated Server Certificate
             X509v3 Subject Key Identifier: 
-                50:DA:1E:62:70:71:C2:72:0D:67:F3:F1:24:A2:E2:9E:31:E1:A3:F0
+                5A:E1:D8:6F:7C:1F:46:73:82:30:4E:68:03:32:C6:AF:AE:EE:B6:28
             X509v3 Authority Key Identifier: 
-                keyid:00:98:A5:D5:E7:C4:55:0E:84:A3:67:FE:66:4A:16:E0:04:15:DD:21
-                DirName:/C=US/ST=NC/L=Raleigh/O=Fedora Project/OU=fedmsg/CN=fedmsg/name=fedmsg/emailAddress=admin@fedoraproject.org
-                serial:8E:EB:28:D8:A9:13:9D:7C
+                keyid:5A:E1:D8:6F:7C:1F:46:73:82:30:4E:68:03:32:C6:AF:AE:EE:B6:28
 
-            X509v3 Extended Key Usage: 
-                TLS Web Server Authentication
-            X509v3 Key Usage: 
-                Digital Signature, Key Encipherment
-    Signature Algorithm: sha1WithRSAEncryption
-        e8:01:80:e4:eb:85:87:d5:be:94:80:e4:e5:d2:df:e8:c6:b5:
-        38:1e:69:31:c0:73:1e:00:60:7b:f4:5f:5c:cb:68:d0:94:d0:
-        bd:9c:ea:33:bd:a1:3a:83:cd:cf:29:57:8b:95:02:d5:bd:fb:
-        fc:33:d8:a8:da:05:84:67:e2:f4:8b:ba:53:e3:e4:67:0d:52:
-        c1:ce:9f:a9:63:ca:86:59:bd:fa:e7:9d:48:8f:b1:e1:fe:a0:
-        a5:6b:0d:6f:03:9c:30:26:80:1b:90:30:e6:a2:ca:4c:29:1e:
-        74:98:5d:cf:34:fb:d7:fa:7a:3a:af:51:96:87:02:dc:18:0a:
-        fb:83
+            X509v3 Basic Constraints: critical
+                CA:TRUE
+    Signature Algorithm: sha256WithRSAEncryption
+         7e:29:8d:20:8b:78:99:d9:cb:3e:a3:64:57:c3:fe:18:16:af:
+         c7:95:66:1f:df:85:a2:e6:8a:8f:a1:39:d5:c9:53:e8:31:a1:
+         bb:e0:3d:44:b5:89:0f:9d:ae:99:4f:44:24:22:25:d2:74:f7:
+         0d:47:61:b8:c9:40:db:28:ca:ad:bd:e5:19:4e:6d:2e:0c:9d:
+         8a:58:5d:f8:e3:e5:98:64:2a:de:1f:03:9b:63:3d:e2:82:48:
+         97:dc:fe:44:c0:08:d9:85:66:e7:bb:23:96:fa:0d:9f:39:54:
+         4a:b6:03:ef:d1:62:fa:fe:31:96:d3:34:da:89:62:81:50:51:
+         e0:ba:12:03:fa:0a:71:89:28:ca:1b:c8:3f:46:bc:24:0c:06:
+         3e:01:49:05:cb:ab:bc:85:42:57:95:82:4b:00:ae:f4:bf:58:
+         09:f7:fb:09:8e:19:11:2c:73:a4:10:54:69:be:bd:4d:e0:5c:
+         91:05:a1:ba:2b:64:b5:4a:27:30:e3:14:a9:ac:17:37:17:2e:
+         87:47:c1:b3:db:a9:c2:12:ef:fd:1c:ad:82:78:c6:b4:8b:c5:
+         92:73:53:06:1b:97:93:66:97:c0:ad:22:ff:31:43:cb:c3:56:
+         dd:2d:70:2e:d1:99:1b:53:0b:e3:97:b1:09:c2:71:aa:e1:52:
+         56:71:65:28:fe:d1:b6:bc:8f:73:63:a9:4c:db:b6:3c:66:03:
+         13:69:83:1b:95:ee:e4:2b:f0:49:84:a8:b1:c1:ce:3a:d5:fa:
+         c6:9e:cc:48:e8:be:9a:44:29:56:c0:88:bc:63:fa:35:14:d1:
+         df:10:62:b2:8f:d6:b4:42:8b:4f:37:cd:06:c1:ff:29:24:4a:
+         1d:23:1b:4c:e3:45:1f:6b:39:18:70:33:1b:95:0a:78:b2:ac:
+         3b:14:aa:e7:99:1f:17:fb:8c:f5:4c:8a:66:c5:86:e4:fc:ed:
+         80:ec:0c:80:38:cf:01:27:f4:6e:7c:c8:3e:26:2f:f1:f1:f0:
+         d6:3b:5f:b1:8d:28:f6:ab:58:06:d6:3d:a3:c2:41:e8:2f:2c:
+         9d:0d:44:3b:3b:37:8f:fb:c6:3d:e7:c7:81:be:c7:be:ae:1f:
+         92:7e:87:c4:5b:6b:85:da:18:ca:bf:6b:2c:f0:c8:c3:72:bc:
+         bb:be:62:e9:93:c5:00:12:e4:49:81:0e:3f:22:a4:ad:76:a0:
+         0d:40:a6:f1:ac:04:f9:69:38:a4:9e:d8:d5:de:77:5b:37:56:
+         b2:80:9a:62:aa:f3:27:2a:43:c3:51:47:43:e4:df:f0:15:cc:
+         2f:bf:d2:72:51:ca:ca:30:eb:58:7c:d7:d9:e4:30:5b:7b:18:
+         a6:f1:b1:69:3c:4e:76:48
 -----BEGIN CERTIFICATE-----
-MIIEKzCCA5SgAwIBAgIBATANBgkqhkiG9w0BAQUFADCBoDELMAkGA1UEBhMCVVMx
-CzAJBgNVBAgTAk5DMRAwDgYDVQQHEwdSYWxlaWdoMRcwFQYDVQQKEw5GZWRvcmEg
-UHJvamVjdDEPMA0GA1UECxMGZmVkbXNnMQ8wDQYDVQQDEwZmZWRtc2cxDzANBgNV
-BCkTBmZlZG1zZzEmMCQGCSqGSIb3DQEJARYXYWRtaW5AZmVkb3JhcHJvamVjdC5v
-cmcwHhcNMTIwNzE1MjExODUyWhcNMjIwNzEzMjExODUyWjCBoDELMAkGA1UEBhMC
-VVMxCzAJBgNVBAgTAk5DMRAwDgYDVQQHEwdSYWxlaWdoMRcwFQYDVQQKEw5GZWRv
-cmEgUHJvamVjdDEPMA0GA1UECxMGZmVkbXNnMQ8wDQYDVQQDEwZmZWRtc2cxDzAN
-BgNVBCkTBmZlZG1zZzEmMCQGCSqGSIb3DQEJARYXYWRtaW5AZmVkb3JhcHJvamVj
-dC5vcmcwgZ8wDQYJKoZIhvcNAQEBBQADgY0AMIGJAoGBALGiXNcQaoOkHWVmUYD1
-ci0Vr3eSrR2tgNjI4u7PEDsHAh5PGMRH1oKNU//CG3JJ/mL+aCbwkY/X4J64Rpbv
-70sH9CEQCdmQkqYlOinKtuhpBt+6p3OxccNsHNN91v86zhW3vHv8bGvWb6f94ChZ
-B0m/Zpw+H1+4hcRhUflnAECnAgMBAAGjggFxMIIBbTAJBgNVHRMEAjAAMBEGCWCG
-SAGG+EIBAQQEAwIGQDA0BglghkgBhvhCAQ0EJxYlRWFzeS1SU0EgR2VuZXJhdGVk
-IFNlcnZlciBDZXJ0aWZpY2F0ZTAdBgNVHQ4EFgQUUNoeYnBxwnINZ/PxJKLinjHh
-o/AwgdUGA1UdIwSBzTCByoAUAJil1efEVQ6Eo2f+ZkoW4AQV3SGhgaakgaMwgaAx
-CzAJBgNVBAYTAlVTMQswCQYDVQQIEwJOQzEQMA4GA1UEBxMHUmFsZWlnaDEXMBUG
-A1UEChMORmVkb3JhIFByb2plY3QxDzANBgNVBAsTBmZlZG1zZzEPMA0GA1UEAxMG
-ZmVkbXNnMQ8wDQYDVQQpEwZmZWRtc2cxJjAkBgkqhkiG9w0BCQEWF2FkbWluQGZl
-ZG9yYXByb2plY3Qub3JnggkAjuso2KkTnXwwEwYDVR0lBAwwCgYIKwYBBQUHAwEw
-CwYDVR0PBAQDAgWgMA0GCSqGSIb3DQEBBQUAA4GBAOgBgOTrhYfVvpSA5OXS3+jG
-tTgeaTHAcx4AYHv0X1zLaNCU0L2c6jO9oTqDzc8pV4uVAtW9+/wz2KjaBYRn4vSL
-ulPj5GcNUsHOn6ljyoZZvfrnnUiPseH+oKVrDW8DnDAmgBuQMOaiykwpHnSYXc80
-+9f6ejqvUZaHAtwYCvuD
+MIIGIzCCBAugAwIBAgIUMSIyZBVPNORqWy7AMKcbelQ4B4QwDQYJKoZIhvcNAQEL
+BQAwgaAxCzAJBgNVBAYTAlVTMQswCQYDVQQIDAJOQzEQMA4GA1UEBwwHUmFsZWln
+aDEXMBUGA1UECgwORmVkb3JhIFByb2plY3QxDzANBgNVBAsMBmZlZG1zZzEPMA0G
+A1UEAwwGZmVkbXNnMQ8wDQYDVQQpDAZmZWRtc2cxJjAkBgkqhkiG9w0BCQEWF2Fk
+bWluQGZlZG9yYXByb2plY3Qub3JnMB4XDTIxMDMyNTEzMjYxNloXDTMxMDMyMzEz
+MjYxNlowgaAxCzAJBgNVBAYTAlVTMQswCQYDVQQIDAJOQzEQMA4GA1UEBwwHUmFs
+ZWlnaDEXMBUGA1UECgwORmVkb3JhIFByb2plY3QxDzANBgNVBAsMBmZlZG1zZzEP
+MA0GA1UEAwwGZmVkbXNnMQ8wDQYDVQQpDAZmZWRtc2cxJjAkBgkqhkiG9w0BCQEW
+F2FkbWluQGZlZG9yYXByb2plY3Qub3JnMIICIjANBgkqhkiG9w0BAQEFAAOCAg8A
+MIICCgKCAgEA0W2U528EQyHlBZBs5+vRberuv90N9Mfn2qXZlw1MnipMxP+2GTqp
+mxlQAWDrcoKpkBOpMriCA3A/mWyP3rtYFzcm3phJ7brfJUsU2cEr8iGLtAEv5OFJ
+5c4ub1q1w1/nCcH3Z1vlS41ss7KU2pGvwwe7Fg007rzCVdBo387kgU908jtXel2Z
+F3XRyMCQ+y/Em4bIlIzC1dXXK2rIPkmITGxSUwqZ/WQaEIvhkCb1q9na8i59rOh6
+BvkpiAu5+K21jETCZWZWhgeJnixi0aXA+nlOT2vH4o3EI7eD+czm5gh8cPFsbEa1
+nKQv0nwOLuT2dFOmml8gkju9NPSJlgY7JZTNUZ8st49MRw6KgFbk1VdSIzZyVwYc
+EE1p4pJV/80u71LBMSG3+kHH8le9cd5G65RFQD8qHi92uaNsKyxla7dairfQDEMF
+BhBeQDnRk7hRfFcHV+prdZRsbQ6g40C4v7sKNusiQoglp5DU9g+S4AUj+ZMJHEQ1
+FNdku3nxZmCOmmsP2P8Zk5vJlRaGeYsEif8sd3TR3VhnCP7IqRPBWboXYWJ2Y7Xo
+R5YzmM49nCdqm13kipzVMYkHRUHztZobVuKFNu/cT7lngDQMSXf48Eezhm5oMz1V
+DA/wKJ/2RpgWN8A3LqV8bzOuAFDfWjn1gM7qOWAgm5itMHsU4RRNqHcCAwEAAaNT
+MFEwHQYDVR0OBBYEFFrh2G98H0ZzgjBOaAMyxq+u7rYoMB8GA1UdIwQYMBaAFFrh
+2G98H0ZzgjBOaAMyxq+u7rYoMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQEL
+BQADggIBAH4pjSCLeJnZyz6jZFfD/hgWr8eVZh/fhaLmio+hOdXJU+gxobvgPUS1
+iQ+drplPRCQiJdJ09w1HYbjJQNsoyq295RlObS4MnYpYXfjj5ZhkKt4fA5tjPeKC
+SJfc/kTACNmFZue7I5b6DZ85VEq2A+/RYvr+MZbTNNqJYoFQUeC6EgP6CnGJKMob
+yD9GvCQMBj4BSQXLq7yFQleVgksArvS/WAn3+wmOGREsc6QQVGm+vU3gXJEFobor
+ZLVKJzDjFKmsFzcXLodHwbPbqcIS7/0crYJ4xrSLxZJzUwYbl5Nml8CtIv8xQ8vD
+Vt0tcC7RmRtTC+OXsQnCcarhUlZxZSj+0ba8j3NjqUzbtjxmAxNpgxuV7uQr8EmE
+qLHBzjrV+saezEjovppEKVbAiLxj+jUU0d8QYrKP1rRCi083zQbB/ykkSh0jG0zj
+RR9rORhwMxuVCniyrDsUqueZHxf7jPVMimbFhuT87YDsDIA4zwEn9G58yD4mL/Hx
+8NY7X7GNKParWAbWPaPCQegvLJ0NRDs7N4/7xj3nx4G+x76uH5J+h8Rba4XaGMq/
+ayzwyMNyvLu+YumTxQAS5EmBDj8ipK12oA1ApvGsBPlpOKSe2NXed1s3VrKAmmKq
+8ycqQ8NRR0Pk3/AVzC+/0nJRysow61h819nkMFt7GKbxsWk8TnZI
 -----END CERTIFICATE-----
diff --git a/fedora_messaging/tests/fixtures/key.pem b/fedora_messaging/tests/fixtures/key.pem
index 2c449b0..31aa0f7 100644
--- a/fedora_messaging/tests/fixtures/key.pem
+++ b/fedora_messaging/tests/fixtures/key.pem
@@ -1,16 +1,52 @@
 -----BEGIN PRIVATE KEY-----
-MIICdQIBADANBgkqhkiG9w0BAQEFAASCAl8wggJbAgEAAoGBALGiXNcQaoOkHWVm
-UYD1ci0Vr3eSrR2tgNjI4u7PEDsHAh5PGMRH1oKNU//CG3JJ/mL+aCbwkY/X4J64
-Rpbv70sH9CEQCdmQkqYlOinKtuhpBt+6p3OxccNsHNN91v86zhW3vHv8bGvWb6f9
-4ChZB0m/Zpw+H1+4hcRhUflnAECnAgMBAAECgYAfnrlMBz4OpN4euLH1dLuYTu3n
-KRLr34C6FjhfLjjDaSH1ZzsPZYJHxwSW2GSuWYGQ+ZxsohInOtRfLIpUP+2mb1SU
-8j0cIlrvYiussHJZ0774OYwW6iRsu9qW14+6T53hY0lCzJq5HRgh1OD95TYDe4D7
-ano1dcAgQ/43ca3ZYQJBAOitxcE5oy2N0zcLlpjV/l12tb3Rcdhigz3phdpj2Hk8
-DI6fLMFynbvFTlHaFjZZVVzsk93Ho3F7Z0sIZ65a1M8CQQDDcDhPmiJMN6b5WaeD
-Lj1BTwjXN4RHUj+xC8nh+l+xjAuBRY2ZNtwbx3jEO2DHWaFQmp8ZsN+yOX/MztOD
-EvypAkAg5qmG6BKF138dUnsqA2BjcOzD+YDCmzN5prKxrdqnUfFonWNju0RIj97C
-PjdT2g/zAtTp0FfxLQTvHDIk+l0LAkBdZw9yc656kIpbVw6osfuSb8xA1wZ/833/
-BixItJjxSIXjFTOUCtTs7cr4tYwOGfbvh8YLyC1Qhwxd/Ga3mOZpAkBJYHyf3bCH
-wEW4PqHOyGUl8yiSNEdhY9VDHec2LT7++oFtlKk+sv1ekzA7uEeygEiKp2WfjNOW
-zmdI0NoDbN+c
+MIIJQwIBADANBgkqhkiG9w0BAQEFAASCCS0wggkpAgEAAoICAQDRbZTnbwRDIeUF
+kGzn69Ft6u6/3Q30x+fapdmXDUyeKkzE/7YZOqmbGVABYOtygqmQE6kyuIIDcD+Z
+bI/eu1gXNybemEntut8lSxTZwSvyIYu0AS/k4Unlzi5vWrXDX+cJwfdnW+VLjWyz
+spTaka/DB7sWDTTuvMJV0GjfzuSBT3TyO1d6XZkXddHIwJD7L8SbhsiUjMLV1dcr
+asg+SYhMbFJTCpn9ZBoQi+GQJvWr2dryLn2s6HoG+SmIC7n4rbWMRMJlZlaGB4me
+LGLRpcD6eU5Pa8fijcQjt4P5zObmCHxw8WxsRrWcpC/SfA4u5PZ0U6aaXyCSO700
+9ImWBjsllM1Rnyy3j0xHDoqAVuTVV1IjNnJXBhwQTWniklX/zS7vUsExIbf6Qcfy
+V71x3kbrlEVAPyoeL3a5o2wrLGVrt1qKt9AMQwUGEF5AOdGTuFF8VwdX6mt1lGxt
+DqDjQLi/uwo26yJCiCWnkNT2D5LgBSP5kwkcRDUU12S7efFmYI6aaw/Y/xmTm8mV
+FoZ5iwSJ/yx3dNHdWGcI/sipE8FZuhdhYnZjtehHljOYzj2cJ2qbXeSKnNUxiQdF
+QfO1mhtW4oU279xPuWeANAxJd/jwR7OGbmgzPVUMD/Aon/ZGmBY3wDcupXxvM64A
+UN9aOfWAzuo5YCCbmK0wexThFE2odwIDAQABAoICAGD1wQjdRzXljWjkEufVMCMZ
+r+u4Zmu72cZLHtBmYeCgaOaUFTTdCgU61OWuf5OGlwaTc7R95AO/hlJ8BqfySLiG
+2kIYc7HS4NkdR8wBOV1i1pFCM3BeNZ5ltO3tK1YHIqpL2UVLMH+oPHl95pjC9oLv
+I1j0SFt3nOmqEcP8v6J2rQgxHTEUAHryG2w8a2NAHGB9KIROHy5tUK9TQt5DoQzw
+4bR8pgrAu6iJeqcIwpkLxeh2Sc0x3L05wlawRyuxEOqeMvpLRSfMeZR3rFCv0waj
+/vOaTPLA+9wlbL7MdOJvb13DCJWgrxiUk2d58/8WFlyJFgtGm4fYjKLlzIVe4SaQ
+KXb6/Mvdwu2/90JIFiyOnQhT42e6324kUhWXldRdxDyvV1xE0WbCWxtvb82mTWHD
+io26pZX7uLdq5PlPrNi+VRRh3hofTdTugVxeZO/iSwMvH2BmYmF92usmN9MLNJH+
+nR6ZtBAR0ClnWhDl7XlrIzTNbHo/4E16PA8WP9wRAguIgluXhgrbrrS8uAVTBfeI
+Wi7MIQ+kFnrQZ1eNpWyORFR8LEepHmFtAn7/HDL2u48sW+ykqYBCDO5/HCBaxW7T
+OL/mR/Kmyosw94DmkGqPgaVehfTl8heAUgmol3g9D8T+TenFfl9QZR1/EXeIFSpN
+NOe3eKcikcdx4Ut6cCphAoIBAQD4othHqP2XMRcHHEtqiuDbXXFPH+Vr7SFe0k81
+xrcFYrZ02k/WbsqqYrHrs9WcR005nIIOvvbwwONp0kiOEDmnzc9nSIi1k65ukvvG
+A5IXHf3Q5zjDXhTMatPojlr7lP6sgl395covpA33J0tegmAjz8rF2YAWvMytxdUe
+lCm4/ilchrciy7/qzqlaaFRYrMuaQOpTiiU5ixt7jULp8pSziWp9hJ17ADc0PxvY
+h1GDZKXE6c31dYOuXxYhkfzDcdi4dW6/hJgGKIuyj7FT2th+5ilgFOrkKytWqLnp
+fFL4lMep8z32R/Db+OrOOyBNJCS9YpjAKxwVu4GmmboPBnW5AoIBAQDXoXZAArLG
+qGVYIbr2GdPGcUUbcvJmxkDkf9WP7hFRa2BwytJGQpd0ao1L90aWCmrliC6C30bj
+3rsCkhq8p5KFaw+6i+a6T9hHYQlLCtGpnSYNA466OSr76HsWBEedrq/eZU/zeV8R
+rtAXbh20+eS7k2T6xQBBj/7oHlttnT0NXa3d+Ox0U+GutK+8E95VKRCRwe34DrPx
+YcAQC/c9feOwVhZYAPq/fM5tF70NPdvrqsnPOJbzoKMtLduQijmQorImA/GSowC5
+1WxijNy8CuoJ72zh85w24i8ACW+8oD/Sqx3s5hn/aEpymfN5KrnroXeWWqZO7Ou7
+ok5czWLklCevAoIBAQCF7/NiZv50u9UT14Io2rdWdHfdj+kewpb/ZXuWFH3zQMHZ
+IOFxOi2iq5f8yNuBXEeViWHmX6jW+JGJJ+Y94S+t3YikssYySmPk3PkUKkhAe7+H
+EYfadovtsn6FAGqyrAilFXVcmLTaBR4gzeIj/0ebAwzQl+IQXZRE6BVua5IuPXw6
+g7eT3v57luhT82+8dRkY929DoV98KBud6vLo4+yG6rea9h1/bRZP5s8xTXPzu/ww
+n2J7n9pXzJE3dFUdU+7TLy+BrAPZDW5zukRkT7fgZExVne8LCtZF0I6V4IsxrbQB
+MS9nxElTtjuG7QLxi07fVoXCv3ZZnrCw9o7sGSbBAoIBAQCtJ3BAuKn9Eu7G1dc/
+lr/Tw9Od1jFHTH6BvxGs0ZzAh/xattQiLYlXp9StIvVguR7cEP5gg5F2nL4Js2+a
+/OmRD8cCoyN1i3Z+4zJGO3rfvFz46Fltu0iEH0zwAiHMQqHhcfPA7T7lgT6DAh8C
+3ATMh/D32sh43Xvxw6bhn/mNySM8c47VXyKEW34fBslsE1zF3zgMESp4dGl5uVIh
+4/y59ndc1fSFXmCZhA2BG6KFXZFPQP8exmkid4ewSv8s4Skzk8kiWpjTocm2Xa1H
+p3AG6GfEDg7sl7bsTaeXjguSTWzrdl98FLBr1KqKK0A/MltItqSEDobCFvTcWuXF
+7h7rAoIBAFLUDPSD9/2zKvK2f2aFjB6QMEa6LTMgAcS7gIIHSIAF4FCUFCL1Pf2s
+IW0q51UJ/5yCdWRK5V/ejB/dFgONaAHzxlLDkge8W0hggEGk1AnEQz4G0544JnbI
+dBCJItmWGzACr8R1e3VjFk2RECI662NvEjHCuyt2EHT0yvo6ENeg3g+ClTrLkgsq
+1UIQnpB0sURtL8nDZvLrz71PgJe1juoqPltFipZ46xEZrXckNeo8yS9iMpNUBr1E
+8OaHsDT3bmFm3Ag4mpapSOXQeGqyAHxQVbuYNz/bJO9yuk9MZt6aa2kqrUXMCl7U
+4Q3qvARMSGOnsGcqFHePKK8wOnvFumY=
 -----END PRIVATE KEY-----
